src/smart-components/user/add-user-to-group/add-user-to-group-wizard.js

```diff
@@ -0,0 +1,251 @@
+import React, { useRef, useContext, useState, useEffect } from 'react';
+import { Modal, Button, ModalVariant } from '@patternfly/react-core';
+import FormRenderer from '@data-driven-forms/react-form-renderer/form-renderer';
+import Pf4FormTemplate from '@data-driven-forms/pf4-component-mapper/form-template';
+import { createRows } from '../../group/group-table-helpers';
+import { applyFiltersToUrl, areFiltersPresentInUrl, syncDefaultFiltersWithUrl } from '../../../helpers/shared/filters';
+import PropTypes from 'prop-types';
+import { Alert } from '@patternfly/react-core';
```

Comment:
Can you please clean the imports, e.g. we import stuff from `@patternfly/react-core` on multiple lines